servers/quarkus-server/src/main/java/org/projectnessie/server/authn/NessieHttpAuthenticator.java

```diff
@@ -46,7 +45,6 @@
 @Alternative // @Alternative + @Priority ensure the original HttpAuthenticator bean is not used
 @Priority(1)
 @ApplicationScoped
-@Startup // Initialize and load config on the main thread rather than on event loop threads
```

Comment:
I added this initially to workaround the context class loader problem that were later mitigated by running each test class in a separate JVM. So, forcing startup init is no longer required.

I suspect forcing startup init might interfere with how Quarkus handles security providers. The startup process involves a set/get round trip though a variable in `ElytronSecurityDomainManager`, which is in turn driven by generated code. ATM I'm not sure what the sequence of actions is or even whether it is stable on each run.

Let's go back to on-demand init for the authenticator, which is what Quarkus does for its default `HttpAuthenticator`.